answerking_app/tests/test_orders.py

```diff
@@ -283,15 +289,19 @@ def test_post_invalid_json_returns_bad_request(self):
         actual = response.json()
 
         # Assert
-        self.assertRaises(ParseError)
-        self.assertIn(expected_json_error, actual["detail"])
+        self.assertIsInstance(actual.pop("traceId"), str)
+        self.assertEqual(expected, actual)
         self.assertEqual(response.status_code, 400)
 
     def test_post_invalid_details_returns_bad_request(self):
         # Arrange
-        invalid_post_data: NewOrderAddressType = {"address": "test%"}
-        expected_failure_error: dict[str, list[str]] = {
-            "address": ["Enter a valid value."]
+        invalid_post_data: OrderType = {"address": "test%"}
+        expected: DetailError = {
+            "detail": "Validation Error",
+            "errors": {"address": ["Enter a valid value."]},
```

Comment:
can we highlight the bad character(s)

Reply:
not sure what you mean, if the serializer.error supports it sure, not that I'm aware, this errors are autogenerated by it